aten/src/ATen/WrapDimUtils.h

```diff
@@ -43,9 +43,19 @@ inline int64_t maybe_wrap_dim(
 inline void maybe_wrap_dims_n(
     int64_t* dims,
     int64_t ndims,
-    int64_t dim_post_expr) {
+    int64_t dim_post_expr,
+    bool wrap_scalars = true) {
```

Comment:
Let's extend the comment describing this function to include a description of each parameter and what they do